### ISSUE TITLE
Missing Azure subscription

### ISSUE THREAD

**Issue by Ayo-faks** [microsoft/AzureStorageExplorer#4236]
**Storage Explorer Version**:  1.18.1
**Build Number**:  20210304.5
**Platform/OS**: Windows 10
**Architecture**:  64
**Regression From**:   first time user

## Bug Description ##
 I get a "we didn't find any Microsoft Azure subscriptions associated with your account" whenever I try to sign in to my Azure storage explorer.
## Steps to Reproduce ##
 Open Azure storage explorer, log in and the error is produced.

## Actual Experience ##
 It accepts my login details, authenticates me, and then goes back to the login page. If I cancel the login page, it shows in the Azure storage explorer that I am logged in but gives the error "we didn't find any Microsoft Azure subscriptions associated with your account".

**Comment by MRayermannMSFT** [comment]
@Ayo-faks can you:

1. Try enabling MSAL in the settings? Go to -> Settings (gear icon on the left) -> Application -> Sign-in -> Check the checkbox for Microsoft Authentication Library, restart, and sign in
2. If you see a new sign in dialog appear after finishing one, don't close the new one! Complete it. As of today, Storage Explorer signs you into ALL of the tenants you have access to. Most of the time, doing this requires you entering your credentials more than once.
3. Are you able to see these subscriptions in the portal?

**Comment by CodeChaser** [comment]
Hello,

I wanted to confirm, after having no issues with the "Desktop - Microsoft Azure Storage Explorer"  a few weeks back.  I did encounter this exact problem as well.

1. After removing my account and re-adding, I also experienced the "verify login loop".
2. I found this comment, followed the recommendation of enabling MSAL.
3. I then removed my account again.
4. Closed the application.
5. Relaunched the Explorer
6. Added my subscription account again

This did resolve my problem, I now have full access.  It is really nice to have addition to the Web UI.

Regards,
Tom Kelly

MCT
Azure Developer Associate
